### PR TITLE
missing report fix to not stay in the same function more than x seconds

### PR DESCRIPTION
**Pull request by SoraKenji**

### CODE REVIEW

File: lib/agent/providers/geo/index.js [hunk 1]
```diff
@@ -1,44 +1,35 @@
-"use strict";
+const strategies = require('./strategies');
+const hooks = require('../../hooks');
+const logger = require('../../common').logger.prefix('geo');
 
-var strategies = require('./strategies'),
-    hooks      = require('./../../hooks'),
-    os_name    = process.platform.replace('win32', 'windows').replace('darwin', 'mac'),
-    logger     = require('../../common').logger.prefix('geo');
+const strategiesList = ['native', 'wifi', 'geoip'];
```
Comment (patriciojofre) [inline lib/agent/providers/geo/index.js:5]: by bye geoip?

File: lib/agent/providers/geo/index.js [hunk 2]
```diff
@@ -1,44 +1,35 @@
-"use strict";
+const strategies = require('./strategies');
```
Comment (patriciojofre) [inline lib/agent/providers/geo/index.js:1]: let's take the opportunity to document this file

Comment (SoraKenji) [reply lib/agent/providers/geo/index.js:1]: This was documented in this PR https://github.com/prey/prey-node-client/pull/806

File: lib/agent/providers/geo/index.js (hunk 1 above)
Comment (SoraKenji) [reply lib/agent/providers/geo/index.js:5]: Removing geoip causes some troubles, so it may be best to remove it later.

**Pull request merged**